web-lib-funcs.pl

```diff
@@ -2809,6 +2838,8 @@ sub complete_http_download
 	$header{lc($1)} = $2;
 	push(@headers, [ lc($1), $2 ]);
 	}
+$$response_headers = \%header
```

Comment:
Instead of using $$ , can't we just pass $response_headers around and then do `%$response_headers = %header` ?

Reply:
> Instead of using $$ , can't we just pass $response_headers around

Yes, check this patch https://github.com/webmin/webmin/commit/72c5e391212c26830ee7dd89b12e9e0db1a746a9.

> and then do `%$response_headers = %header` ?

No, it won't work like this!

Reply:
Thanks!